src/OrbitQt/CaptureOptionsDialog.ui

```diff
@@ -20,411 +20,450 @@
             <property name="text">
-             <string>Callstack sampling period (ms):</string>
+             <string>Enable callstack sampling</string>
+            </property>
+            <property name="checked">
+             <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
-           <widget class="QDoubleSpinBox" name="samplingPeriodMsDoubleSpinBox">
-            <property name="decimals">
-             <number>1</number>
-            </property>
-            <property name="minimum">
-             <double>0.100000000000000</double>
-            </property>
-            <property name="maximum">
-             <double>10.000000000000000</double>
-            </property>
-            <property name="singleStep">
-             <double>0.100000000000000</double>
-            </property>
-            <property name="value">
-             <double>1.000000000000000</double>
-            </property>
-           </widget>
+           <layout class="QHBoxLayout" name="samplingPeriodMsHorizontalLayout">
+            <item>
+             <widget class="QLabel" name="samplingPeriodMsLabel">
+              <property name="text">
+               <string>Callstack sampling period (ms):</string>
+              </property>
+             </widget>
+            </item>
+            <item>
+             <widget class="QDoubleSpinBox" name="samplingPeriodMsDoubleSpinBox">
+              <property name="decimals">
+               <number>1</number>
+              </property>
+              <property name="minimum">
+               <double>0.100000000000000</double>
+              </property>
+              <property name="maximum">
+               <double>10.000000000000000</double>
+              </property>
+              <property name="singleStep">
+               <double>0.100000000000000</double>
+              </property>
+              <property name="value">
+               <double>1.000000000000000</double>
+              </property>
+             </widget>
+            </item>
+            <item>
+             <spacer name="samplingPeriodMsHorizontalSpacer">
+              <property name="orientation">
+               <enum>Qt::Horizontal</enum>
+              </property>
+              <property name="sizeHint" stdset="0">
+               <size>
+                <width>40</width>
+                <height>20</height>
+               </size>
+              </property>
+             </spacer>
+            </item>
+           </layout>
           </item>
           <item>
-           <spacer name="samplingPeriodMsHorizontalSpacer">
-            <property name="orientation">
-             <enum>Qt::Horizontal</enum>
-            </property>
-            <property name="sizeHint" stdset="0">
-             <size>
-              <width>40</width>
-              <height>20</height>
-             </size>
+           <widget class="QGroupBox" name="unwindingMethodGroupBox">
```

Comment:
I don't have an alternative, but I don't really like the nested group too much. Maybe it's only me but maybe someone has an idea on this.

Reply:
It's probably only me...